AdamantShared/AdamantResources.swift

```diff
@@ -87,7 +91,7 @@ struct AdamantResources {
     //    static let ethereumExplorerAddress = "https://ropsten.etherscan.io/tx/" // Testnet
     
     // MARK: LSK
-    static let liskExplorerAddress = "https://explorer.lisk.io/tx/"
+    static let liskExplorerAddress = "https://liskscan.com/transaction/" //"https://explorer.lisk.io/tx/"
```

Comment:
Remove commented https://explorer.lisk.io/tx/ and https://testnet-explorer.lisk.io/tx